.gitmodules

```diff
@@ -10,3 +10,6 @@
 [submodule "thirdparty/libbcrypt"]
 	path = thirdparty/libbcrypt
 	url = https://github.com/trusch/libbcrypt.git
+[submodule "thirdparty/docker-utils"]
+	path = thirdparty/docker-utils
+	url = https://github.com/TheNoim/utils
```

Comment:
This is fine for now, but maybe create a PR into the main repo with your additions and make a TODO note that we should really point at the main `lcdr` when those changes are merged?

Reply:
Added a TODO to track this effort: https://github.com/TheNoim/DarkflameServer/pull/6